### PR TITLE
List Vidulum (VDL)

### PR DESCRIPTION
**Pull request by Corey-Code**
- Official project URL: https://Vidulum.app
- Official block explorer URL: https://explorer.vidulum.app

### CODE REVIEW

File: assets/src/main/java/bisq/asset/coins/Vidulum.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package bisq.asset.coins;
+
+import bisq.asset.AddressValidationResult;
+import bisq.asset.AddressValidator;
+import bisq.asset.Coin;
+
+public class Vidulum extends Coin {
+
+    public Vidulum() {
+        super("Vidulum", "VDL", new VidulumAddressValidator());
```
Comment (blabno) [inline assets/src/main/java/bisq/asset/coins/Vidulum.java]: Use existing Regex validator

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

**Review by Corey-Code — COMMENTED** [review]
fixed as requested

File: assets/src/main/java/bisq/asset/coins/Vidulum.java (hunk 1 above)
Comment (Corey-Code) [reply assets/src/main/java/bisq/asset/coins/Vidulum.java]: done

**Review by Corey-Code — COMMENTED** [review]
fix test

**Comment by blabno** [comment]
Tests are failing

**Comment by ManfredKarrer** [comment]
Close as repeated test fails. Did you ever run the tests?

**Pull request closed without merging**